Source/Core/EventEmitter.swift

```diff
@@ -19,13 +19,13 @@ class ReceiverHandle<Parameters>: Hashable {
 
     fileprivate init(object: AnyObject, handler: @escaping (AnyObject) -> (Parameters) -> Void) {
         self.object = object
```

Comment:
😬

Reply:
This is normal for an init in Swift.
or like ‘guard let self = self else {return}’ in closures.